compute_tools/src/spec.rs

```diff
@@ -678,6 +678,13 @@ pub fn handle_extensions(spec: &ComputeSpec, client: &mut Client) -> Result<()>
             let query = "CREATE EXTENSION IF NOT EXISTS pg_stat_statements";
             info!("creating system extensions with query: {}", query);
             client.simple_query(query)?;
+            // since customers can not use superuser we grant execute to neon_superuser
+            let query = "GRANT EXECUTE ON FUNCTION pg_stat_statements_reset() TO neon_superuser";
```

Comment:
this code will be executed only for users who have this feature flag. we didn’t expose it for the users (you might see the Insights tab in the console, this part of code is used for this particular feature). 

It means that your changes will not help for general use case

Reply:
I marked this PR as draft and did not yet assign a reviewer, this is WIP

Reply:
New solution doesn't modify spec.rs